### DISCUSSION TITLE
Help re-routing

### DISCUSSION THREAD

**Discussion by jl94x4** [lostb1t/replex#71] [Q&A]
I've looked at the readme and it mentions rerouting

/video/:/transcode/universal/session
/library/parts

But I'm not sure how to do this, can anyone help me with this?

My setup is using Nginx Proxy Manager to create plex.domain.com on port 80 which is what I use for REPLEX and my docker command is as follows;

`docker run --rm -it -p 90:80 -e REPLEX_HOST="http://ip:32400" -e REPLEX_AUTO_SELECT_VERSION=true -e REPLEX_DISABLE_RELATED=true -e REPLEX_CACHE_ROWS=true -e REPLEX_CACHE_ROWS_REFRESH=true ghcr.io/lostb1t/replex:latest`

Any help is greatly appreciated.

**Comment by lostb1t** [comment]
That depends on the reverse proxy your are using for nginx proxy manager you should be able to add custom nginx rules with https://nginxproxymanager.com/advanced-config/#custom-nginx-configurations

**Comment by PierreDurrr** [comment]
> I've looked at the readme and it mentions rerouting
> 
> /video/:/transcode/universal/session /library/parts
> 
> But I'm not sure how to do this, can anyone help me with this?
> 
> My setup is using Nginx Proxy Manager to create plex.domain.com on port 80 which is what I use for REPLEX and my docker command is as follows;
> 
> `docker run --rm -it -p 90:80 -e REPLEX_HOST="http://ip:32400" -e REPLEX_AUTO_SELECT_VERSION=true -e REPLEX_DISABLE_RELATED=true -e REPLEX_CACHE_ROWS=true -e REPLEX_CACHE_ROWS_REFRESH=true ghcr.io/lostb1t/replex:latest`
> 
> Any help is greatly appreciated.

Did you figured out ? I'm looking at this too...

**Comment by lostb1t** [comment]
here's an example from the top of my head. Untested but should point you in the right direction

```nginx
upstream plex {
    server 127.0.0.1:32400;
    keepalive 8;
}

upstream replex {
    server 127.0.0.1:3001;
    keepalive 8;
}


server {
    listen 0.0.0.0:80;


    location / {
        proxy_pass http://replex;
    }

    location /video/:/transcode/universal/session {
        proxy_pass http://plex;
    }

    location /library/parts {
        proxy_pass http://plex;  
    }
 }

```

**Reply by jl94x4** [reply]
Can this be passed through nginx proxy manager?

**Comment by jl94x4** [comment]
![image](https://github.com/lostb1t/replex/assets/18123855/a9201e00-b5af-4613-8799-b3b724a8e73e)

I tried inputting it here but it broke the proxy.

And also here;

![image](https://github.com/lostb1t/replex/assets/18123855/d7b16c69-a9d6-441f-a37a-42f1081e8192)

**Comment by jl94x4** [comment]
This works and loads, does this look right?

![image](https://github.com/lostb1t/replex/assets/18123855/46bf4019-8730-42af-abc3-dd8508208a2b)

**Reply by lostb1t** [reply]
Yeah looks good

**Comment by PierreDurrr** [comment]
> This works and loads, does this look right?
> 
> ![image](https://user-images.githubusercontent.com/18123855/266747255-46bf4019-8730-42af-abc3-dd8508208a2b.png)

Did you get it working ?
